presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -75,6 +75,7 @@
 import com.facebook.presto.operator.scalar.CombineHashFunction;
```

Comment:
Not really, because separating the different elements of the implementation would mean that each individual commit would have a partially working version of grouping and that seems wrong.